plugins/actions/pipeline/src/main/java/org/apache/hop/workflow/actions/pipeline/ActionPipeline.java

```diff
@@ -624,7 +624,6 @@ public Result execute(Result result, int nr) throws HopException {
           pipeline.waitUntilFinished();
 
           if (parentWorkflow.isStopped() || pipeline.getErrors() != 0) {
-            pipeline.stopAll();
```

Comment:
@mattcasters , could you double check this change. (HOP-2954) 
When debugging I noticed transforms.stopAll() was called twice, once via pipeline.execute()
and then a second time by triggering this line, because everything was already stopped it got stuck in a loop.